### PR TITLE
Add smooth scrolling and link to MLH logo

### PR DESCRIPTION
**Pull request by joeytepp**
## Proposed Change
Add smooth scrolling and a link on the MLH logo on the `Landing` component

Fixes: #207

### How did you do this?
Added some css for smooth scrolling, turned the QHacks logo to a link to the top of the page and the MLH logo to a link to their website

### Why are you choosing this approach?
To improve UX!

### Any open questions you want to ask code reviewers?
Nope

### List of changes:

  - Add link to MLH logo on `Landing`
  - Add link to QHacks logo in `Header`
  - Update tests

## Pull Request Checklist:

  - [x] My submission passes all tests.
  - [x] I have lint my code locally prior to submission.
  - [x] I have written new unit tests for my core changes (where applicable).
  - [x] I have written browser integration tests for my core changes (where application).
  - [x] I have referenced all useful information (issues, tasks, etc) in the pull request.

### CODE REVIEW

**Comment by RobertWSaunders** [comment]
Deploy preview for *qhacks-website* ready!

Built with commit bb57aac58e6c3bdf61e0823225a940921cf66c81

https://deploy-preview-208--qhacks-website.netlify.com

**Comment by joeytepp** [comment]
The smooth scrolling seems to desktop only, however it fails gracefully on mobile!

**Comment by RobertWSaunders** [comment]
Are you looking for a solution to the mobile issue or no?

**Comment by joeytepp** [comment]
No, I don't really see the benefit of smooth scrolling with the mobile menu, you can't really see it since the menu covers the screen

**Comment by joeytepp** [comment]
The scrolling is barely noticeable when you use the website in desktop with a smaller screen

**Comment by RobertWSaunders** [comment]
Fair enough, I'll give this a review then! 😄

File: src/pages/index.css [hunk 1]
```diff
@@ -40,6 +40,7 @@ body {
   font-size: 15px;
   font-weight: 400;
   color: #000;
+  scroll-behavior: smooth;
```
Comment (RobertWSaunders) [inline src/pages/index.css]: I actually didn't know about this CSS rule, nice! 👍

File: src/components/Menu.js [hunk 2]
```diff
@@ -11,18 +11,20 @@ const Menu = (props) => {
...
-          paddingBottom: "20px",
-          transition: "0.5s",
-          "@media(min-width:820px)": {
-            opacity: props.imgCss ? "1" : "0"
-          }
-        }}
-        src={colored}
-        alt="QHacks Crown"
-      />
+      <a href="/#">
```
Comment (RobertWSaunders) [inline src/components/Menu.js]: Nice!

File: src/components/Landing.js [hunk 3]
```diff
@@ -108,15 +109,17 @@ const Landing = () => (
...
-          <img
-            data-cy="mlh-banner"
-            src={banner}
-            css={{
-              width: "189px",
-              "@media(max-width: 820px)": { width: "132px" }
-            }}
-            alt="MLH Logo"
-          />
+          <a href="https://mlh.io" target="_blank" rel="noopener noreferrer">
```
Comment (RobertWSaunders) [inline src/components/Landing.js]: Nice!

File: src/components/Landing.js [hunk 4]
```diff
@@ -29,6 +29,7 @@ const loginButtonCSS = {
 
 const Landing = () => (
   <div
+    id=""
```
Comment (RobertWSaunders) [inline src/components/Landing.js]: Whats this for?

Comment (joeytepp) [reply src/components/Landing.js]: So that `qhacks.io/#` links to the top of the site!

Comment (joeytepp) [reply src/components/Landing.js]: Can change this to something more descriptive, but I think this looks best

Comment (RobertWSaunders) [reply src/components/Landing.js]: Cool cool!

**Pull request merged**